src/highrise/__main__.py

```diff
@@ -58,7 +60,11 @@ async def main(bot: BaseBot, room_id: str, api_key: str) -> None:
                 async with ClientSession() as session:
                     async with session.ws_connect(
                         environ.get("HR_WEBAPI_URL", "wss://highrise.game/web/webapi"),
-                        headers={"room-id": room_id, "api-token": api_key},
+                        headers={
+                            "room-id": room_id,
+                            "api-token": api_key,
+                            SDK_NAME: version,
```

Comment:
I think we should use the `user-agent` header here, it's standardized and fits the use case well.

Like `highrise-bot-sdk/version`.

Reply:
Changed